_data/email.yml

```diff
@@ -32,12 +32,19 @@ websites:
       hardware: Yes
       doc: https://www.google.com/intl/en-US/landing/2step/features.html
 
-    - name: GMX
+    - name: GMX.com
       url: https://www.gmx.com/
       twitter: gmxmail
       img: gmx.png
       tfa: No
 
+    - name: GMX.net
+      url: https://www.gmx.net/
+      img: gmx.png
```

Comment:
```suggestion
      img: gmx.net.png
```

Reply:
@GeileHirnbude I am not allowed to push changes to your repository, please commit this change so that the travis check runs ok. Thanks!
// Kai